resources/zib/ext-AdvanceDirective.Disorder.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-AdvanceDirective.Disorder" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-AdvanceDirective.Disorder" />
+  <name value="ExtAdvanceDirectiveDisorder" />
+  <title value="ext AdvanceDirective.Disorder" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The Disorder concept of zib AdvanceDirective." />
+  <purpose value="This extension represents the Disorder concept of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) AdvanceDirective-v3.1.1(2020EN)](https://zibs.nl/wiki/AdvanceDirective-v3.1.1(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-advancedirective-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/AdvanceDirective-v3.1.1(2020EN)" />
+    <name value="zib AdvanceDirective-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Consent" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-AdvanceDirective.Disorder" />
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]" />
+      <short value="Disorder" />
+      <definition value="The problem or disorder to which the living will applies." />
+      <alias value="Aandoening" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Problem" />

```

Comment:
Should this be nl-core or not? See remark in #169

Reply:
Yes, this should be core!Thnx.